base.js

```diff
@@ -263,6 +271,38 @@ const WIRE_ADAPTERS_WITH_RESTRICTED_USE = [
         module: 'lightning/uiRecordApi',
         identifier: 'getRecordUi',
     },
+    {
+        module: 'lightning/uiRelatedListApi',
+        identifier: 'getRelatedListsCount',
+    },
```

Comment:
I don't see this export

Reply:
I was going based on the fact that the module `uiRelatedListApi` is exposed per [this js-meta.xml file](https://github.com/salesforce/lightning-components/blob/master/ui-lightning-components/src/main/modules/interop/uiRelatedListApi/uiRelatedListApi.js-meta.xml). Is that an incorrect way to approach this?

Reply:
Should be fixed now!